fitnessllm_dataplatform/stream/strava/entities/enums.py

```
@@ -0,0 +1,43 @@
+    LATLNG = "latlng"
+    ALTITUDE = "altitude"
+    VELOCITY_SMOOTH = "velocity_smooth"
+    HEARTRATE = "heartrate"
+    CADENCE = "cadence"
+    WATTS = "watts"
+    TEMP = "temp"
+    MOVING = "moving"
+    GRADE_SMOOTH = "grade_smooth"
+
+    @classmethod
+    def filter_streams(cls, include=None, exclude=None) -> list:
```

Ruff (0.8.2)

Missing type annotation for function argument `include`

(ANN001)